src/web/mjs/connectors/FuryoSquad.mjs

```diff
@@ -29,10 +29,10 @@ export default class FuryoSquad extends Connector {
     }
 
     async _getChapters(manga) {
-        const request = new Request(new URL(manga.id, this.url), this.requestOptions);
-        const data = await this.fetchDOM(request, 'div.list.fs-chapter-list div.title a');
+        const url = new URL(manga.id, this.url);
+        const data = await this._fetchPOST(url, 'adult=true', 'div.list.fs-chapter-list div.title a');
 
-        return data.map(chapter => {
+        return [...data].map(chapter => {
```

Comment:
May move spread operator into `_fetchPOST`

Reply:
I'll leave it for now. Still playing with the idea of implementing a `Connector.fetchPOST` which will be cleaner and more generic.